plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -309,4 +341,14 @@ public IAdaptable getElement()
     public void setElement(IAdaptable element)
     {
     }
+
+    /**
+     * Return available charsets with first item of system encoding
+     */
+    public static List<String> availableCharsets() {
+        List<String> charsetList = new ArrayList<>();
+        charsetList.add(String.format("%s (%s)", "Default encoding", BaseApplicationImpl.getInstance().getSystemEncoding()));
+        charsetList.addAll(Charset.availableCharsets().keySet());
```

Comment:
I expect this set to be ordered chaotically. Maybe it's better to sort it alphabetically?